examples/pip_requirements/pip_requirements.py

```diff
@@ -45,22 +45,19 @@ def main():
         mlflow.xgboost.log_model(model, artifact_path)
         pip_reqs = get_pip_requirements(run_id, artifact_path)
         assert pip_reqs == ["mlflow", xgb_req], pip_reqs
-        print("Model URI:", mlflow.get_artifact_uri(artifact_path))
```

Comment:
@dbczumar I've reconsidered if we really need these print statements. I don't we need them because we can view the logged artifacts on MLflow UI and just showing the URI is not helpful for users so I've deleted them.